app/src/main/res/values/strings.xml

```diff
@@ -12,7 +12,7 @@
     <string name="wc_about">About</string>
     <string name="wc_location">Location</string>
     <string name="action_search">Search WordCamps</string>
-    <string name="search_wc_hint">Search WordCamps</string>
+    <string name="search_wc_hint"><![CDATA[<font color = #ffffff> Search WordCamps </font>]]></string>
```

Comment:
Don't use CDATA in strings, translator won't know how to translate this. It's best to split the string and only keep the translatable part.